server/fileupdater.mjs

```diff
@@ -319,3 +320,9 @@ function v6cssPieces(properties) {
     }
   }
 }
+
+function v7HolderClickable(properties) {
+  if (properties.clickRoutine && !properties.clickable && properties.type=='holder'){
+    properties.clickable=false;
+  }
```

Comment:
I don't think this should be so specific. It should recreate the old behavior for all cases.

Reply:
well think it should:
- clickable and clickRoutine: no change
- clickable and no clickRoutine: no change. useless, as we discussed, but not problematic
- not clickable and clickRoutine: no change. at this point this is probably intentional and changing would probably break something. but now it is stated that is is set to false. probably will have less confusion.
- not clickable and no clickRoutine: CHANGE to clickable. useless but it will create less confusion if anyone wants to add a clickRoutine and is consistent to the new default. Should be a safe change

I did test the file updater and results were as expected

Reply:
I see what you're saying. Gotta think about it.. :grin: